docker/pypi/wmagent/run.sh

```diff
@@ -1,11 +1,23 @@
 #!/bin/bash
 
 ### Basic initialization wrapper for WMAgent to serve as the main entry point for the WMAgent Docker container
+wmaUser=$(id -un)
+wmaGroup=$(id -gn)
+wmaUserID=$(id -u)
+wmaGroupID=$(id -g)
+export WMA_USER=$wmaUser
+echo "Running WMAgent container with user: $wmaUser (ID: $wmaUserID) and group: $wmaGroup (ID: $wmaGroupID)"
 
+echo "Correcting ownership for WMA_ROOT_DIR: $WMA_ROOT_DIR"
+find $WMA_ROOT_DIR \! \( -user $wmaUser -group $wmaGroup \) -exec chown -f $wmaUser:$wmaGroup '{}' +;
```

Comment:
@amaltaro I have two questions regarding this line:

* why this complicate command instead just:
```
chown -R $wmaUser:$wmaGroup $WMA_ROOT_DIR
```
* why do we actually need this. The whole $WMA_ROOT_DIR is expected to be created with the proper user permissions outside the container. In our case it is done here:  https://github.com/dmwm/CMSKubernetes/blob/f4a43f8a85bb1be3b59b913b59914f5f66014a34/docker/pypi/wmagent/wmagent-docker-run.sh#L59-L63 
And if it is not a `sudo` command it will break and do nothing. 
 
Just to clarify, in the past I have been seeing many times the mount  dir having files with root privileges, and in case this was the driving force behind this command to be moved here, I am pretty sure this is not the right solution. Because those files show up with `root:root` ownership, only when the container was wrongly run and have not switched to the proper user. Then docker engine is creating what is needed for us to run, and docker engine runs with `root`. This was  something which took me  quite a while in the past to block happening.  What we need to do is to get assured that the container is about to be executed at the agent with the same user every time.  

In addition for T0 their account is not sudo, so for them this was failing all the time. (I mean, I've walked this path already)

Reply:
Indeed I saw directories being owned by root yesterday. Let me remove this code then.